api/src/test/java/marquez/db/migrations/V44_3_BackfillJobsWithParentsTest.java

```diff
@@ -35,6 +29,7 @@
 import org.junit.jupiter.api.extension.ExtendWith;
 
 @ExtendWith(MarquezJdbiExternalPostgresExtension.class)
+@FlywayTarget("44") // apply migrations up to and including v44 but not beyond
```

Comment:
_Minor_: Similar comment on commenting on fixing the migration to `44`.

Reply:
Done